example_configs/tandoor_recipes.md

```diff
@@ -0,0 +1,8 @@
+LDAP_AUTH=1
+AUTH_LDAP_SERVER_URI=ldap://lldap:3890/
+AUTH_LDAP_BIND_DN=uid=ro_admin,ou=people,DC=example,DC=com
```

Comment:
Small nit: can you use consistent casing for the DC and ou and so on? Maybe everything in lowercase

Reply:
Done!